src/App.tsx

```diff
@@ -1,9 +1,24 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.scss';
 import { MoviesList } from './components/MoviesList';
 import moviesFromServer from './api/movies.json';
 
 export const App: React.FC = () => {
+  const [query, setQuery] = useState('');
+
+  const visibleMovies = () => {
```

Comment:
If it looks like a duck, swims like a duck, and quacks like a duck, then it should be named a duck.
Function name normally starts with a verb. Those are `get...`, `set...`, `calculate...`, `switch...` and so on